tests/fr/light_HassTurnOn.yaml

```diff
@@ -1,6 +1,23 @@
 language: fr
 tests:
   - sentences:
+      - Allumer les lumières du salon
+      - Allume la lumière du salon
+      - Allumer les lumières dans le salon
+      - Allume toutes les lumières dans le salon
+      - Active la lumière du salon
+      - Lumières dans le salon
+      - Lumière dans le salon
+      - Lumière salon
+      - Éclaire le salon
+    intent:
+      name: HassTurnOn
+      slots:
+        area: salon
+        domain: light
+    response: Lumières allumées
+  - sentences:
```

Comment:
We don't need to duplicate sentences.

Reply:
i have removed unnecessary sentence test